meetings/2022-11-07.md

```diff
@@ -0,0 +1,107 @@
+* eP: Agree that it will add complexity, but we also don't currently have a mechanism for trusted apps / app stores. There could be community / corporate trust anchors. There have been conversations to have a more centralized / review process for apps. So while there is complexity, it also provides a solution (partial?) to a problem that we haven't yet been able to solve.
+* WT: I agree that there could be some advantage. IMO even now developers need tooling to assist them.
+* AC: I agree with both of you. Yes it is more complicated but it also opens up to some features that we will need and don't have at this moment.
+* WT: Where in the spec it says that the leaf entity can't be their own trust anchor?
+* AC: Let me find it... https://openid.net/specs/openid-connect-federation-1_0.html#section-4.8
+* ...: It relates to federation fetch endpoint which has mutually exclusive requirements for trust anchor and leaf entity.
+* WT: Maybe it's just an oversight. 
+* eP: Perhaps we should create an issue for this. We can then get a sense of whether this might be addressed in the federation spec
+* AC: I think I have a good idea about the problem.
+
+ACTION: AC to create issue tracking disjont definition of leaf entity and trust 
+
+* ES: I understand that federation works with the same claims as dynamic registration. Is there a way to add more data? Let's say we have trus anchor at solidproject.org, can it say that all clients registered with it have a flag that it marks it as 'in preview' or untrusted. It would be helpful with any app marketplace to mark apps.
+* AC: Yes you can add metadata. Inside of Entity Configuration there is *metadata* secion. It is generally what we would see in .well-known/openid-configuration for the RP there would be mostly dynamic client registration information. There is also *metadata policy* which can for example define constraints in all entities in the federation, which is kind of nice. It can set high level constraints in the federation which can be well defined in the whole system.
+* ES: People often mix localhost and production URLs together, which I recommend since it says it is the same client. To be able to have a federation server, which says *all documents registered with me are developmen preview*. On the consent pages we can flag that aspec of the application. I think trust will be a big part in the future for any application.
+* AC: I hear practically from all of us that trust is important. What OIDC Federation can give us provides much better foundation for building this trust. We can devise a specific federation which is for development. When you use that app with your pod you will know how much you can trust it. Now everything is a hodge podge.
+* AC: I would like a strawman vote just to gauge opinions.
+* eP: In support of exploring this. Am interested in drafting a PR to the primer, where we can see what form this will take. That will give us an idea for how the spec change would look.
```

Comment:
```suggestion
* eP: I support exploring this. Am interested in drafting a PR to the primer, where we can see what form this will take. That will give us an idea for how the spec change would look.
```